_pages/plugins/motiq.md

```diff
@@ -0,0 +1,27 @@
+---
+mediawiki: MotiQ
```

Comment:
The `mediawiki` tag is for backlinking to the old content from the MediaWiki-based ImageJ wiki. Since this is new content, it should not be used here.